Makefile

```diff
@@ -30,6 +30,10 @@ ifeq ("$(wildcard $(VALKEY_ROOT))","")
     $(error Please provide the VALKEY_ROOT variable pointing to the Valkey source code)
 endif
 
+ifeq ("$(wildcard $(VALKEY_BLOOM_ROOT))","")
+	$(error Please provide the VALKEY_BLOOM_ROOT variable pointing to the valkey-bloom source code)
+endif
```

Comment:
It needs to be possible to build man pages without bloom filter documentation. We could print a warning or something, but then just build the docs and exclude the bf commands from the docs.

If you build and install valkey and valkey-doc without valkey-bloom, you should be able to build man pages without the bloom filter commands.

In a distro (fedora, debian, etc.) all components tend to be separate packages that you install independently, though there are often meta-packages too that bundle multiple packages. The point is that we should try to keep the different components separate for better maintainability and flexibility.